app/views/result/index.html.erb

```diff
@@ -59,61 +59,77 @@
           <% row.each do |side| %>
             <th class="results-column">
               <p class="results-text" >
-                <a href = "<%= "#{link_format(side['link'])}" %>" >
-                  <%= truncate(name(side['link']), :length => 60 ) %>
+
+                <a href = "<%= side.link %>" >
+                  <%= truncate(side.name, :length => 60 ) %>
                 </a>
+
                 <p><%= side['title'] %></p><br>
+
                 <span class="head-info">
-                  <%= head_info_list_maker(side).compact.join(", ") %>
+                  <% head_fields = Searching::HEAD_FIELDS.select { |field| enabled?(field) } %>
```

Comment:
this should be outside the loop.. or better yet in the controller

Reply:
thanks for suggesting this, first use of controller